code/modules/mob/mob_movement.dm

```diff
@@ -339,7 +339,6 @@
 		//Wheelchair pushing goes here for now.
 		//TODO: Fuck wheelchairs.
 		if(istype(mob.pulledby, /obj/structure/bed/stool/chair/office/wheelchair) || istype(mob.pulledby, /obj/structure/janitorialcart))
```

Comment:
you know you can just remove "istype(mob.pulledby, /obj/structure/janitorialcart)" here instead right

also this will mess with wheelchair slowdown too

Reply:
addressed.